app/schemas.py

```diff
@@ -81,13 +81,20 @@ class LocationBase(LocationCreate):
 
 class PriceCreate(BaseModel):
     model_config = ConfigDict(from_attributes=True, arbitrary_types_allowed=True)
+
     product_code: str | None = Field(
         default=None,
         min_length=1,
         pattern="^[0-9]+$",
         description="barcode (EAN) of the product, as a string.",
         examples=["16584958", "8001505005707"],
     )
+    product_name: str | None = Field(
```

Comment:
@raphael0202 in terms of naming, I hesitated with `product_label` and `product_tag`. But there are the existing `category_tag` & `labels_tags` so it could be confusing... or maybe `product_store_label` ? or `product_name_in_store` ?

Reply:
`product_name` sounds good to be!